src/goBuild.ts

```diff
@@ -23,13 +23,18 @@ export function buildCode(buildWorkspace?: boolean) {
 
 	let documentUri = editor ? editor.document.uri : null;
 	let goConfig = vscode.workspace.getConfiguration('go', documentUri);
+	statusBarItem.show();
+	statusBarItem.text = 'Building in progress';

```

Comment:
Lets shorten this to `Building...`
Some users might have isntalled extensions that are already occupying some space in the status bar. Its best if we occupy as less space as possible